.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,68 @@
+name: Run benchmarks
+
+on:
+  workflow_dispatch:
+    inputs:
+      class:
+        description: 'Benchmark class'
+        required: true
+        default: '*'
+      jdk:
+        description: 'JDK version: 8 or 11'
+        required: true
+        default: '8'
```

Comment:
Could we use a fixed JDK version? For example: 8.0.282?

Reply:
Let's just leave it ... according to the semver, the performance should stay basically same for maintenance releases.

Reply:
OK